frontend/server/src/main/resources/log4j.properties

```diff
@@ -17,7 +17,7 @@ log4j.appender.mms_metrics = org.apache.log4j.RollingFileAppender
 log4j.appender.mms_metrics.File = ${METRICS_LOCATION}/mms_metrics.log
 log4j.appender.mms_metrics.MaxFileSize = 100MB
```

Comment:
Nikil mentioned there is a standard rotation size and number for AWS server, better to check and match them